src/anomalib/data/utils/transforms.py

```diff
@@ -162,3 +164,86 @@ def get_transforms(
         transforms = A.Compose(transforms_list, additional_targets={"image": "image", "depth_image": "image"})
 
     return transforms
+
+
+class Denormalize:
```

Comment:
Is this still used? I think it's been deprecated for a while so we could probably remove it (after confirming that it's not used anywhere of course)

Reply:
It is used in some of the jupyter notebooks